Sources/Sentry/SentrySessionTracker.m

```diff
@@ -105,6 +118,9 @@ - (void)stop
         removeObserver:self
                   name:SentryNSNotificationCenterWrapper.willTerminateNotificationName];
 #endif
+    // Reset the `wasDidBecomeActiveCalled` flag to ensure that the next time
+    // `didBecomeActive` is called, it will start a new session.
+    self.wasDidBecomeActiveCalled = NO;
```

Comment:
`m`: I think we should find a better name for `wasDidBecomeActiveCalled` cause it's now set to true in `startSession` even when not calling `didBecomeActive`. `wasStartSessionCalled` is my first shot.